CONTRIBUTING.md

```diff
@@ -545,15 +531,15 @@ https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/maintain
 
    ```sh-session
    mpdev install \
-         --version_meta_file=gs://YOUR-BUCKET/YOUR-COMPANY/YOUR-APP/TRACK/VERSION \
+         --version_meta_file=gs://YOUR-BUCKET/YOUR-COMPANY/TRACK/VERSION \
          --parameters='{"name": "YOUR-APP-INSTANCE", "namespace": "YOUR-NAMESPACE" <APP-SPECIFIC-PARAMETERS> }'
    ```
 
    For example:
 
    ```sh-session
    mpdev install \
-         --version_meta_file=gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark/conjur-open-source:1.6/1.6.1.yaml \
+         --version_meta_file=gs://artifacts.conjur-cloud-launcher-onboard.appspot.com/cyberark:1.6/1.6.1.yaml \
```

Comment:
same as above

Reply:
Yes, I'll switch this to use {version} in place of the `1.6.1`, and in place of the `1.6` I'll use a track/minor version that's derived from the patch {version}.